actors/verifreg/tests/harness/mod.rs

```diff
@@ -140,11 +139,8 @@ impl Harness {
         );
 
         rt.expect_emitted_event(
-            EventBuilder::new()
-                .typ("verifier-balance")
-                .field_indexed("verifier", &verifier_resolved.id().unwrap())
-                .field("balance", &BigIntSer(allowance))
-                .build()?,
+            emit::build_verifier_balance_event(verifier_resolved.id().unwrap(), &None, allowance)
```

Comment:
The event-building code is now untested, since you're using it to verify itself. Yes, it's a bit verbose, but please restore the explicit checking of event contents (helpers like the expect.rs `build_verifreg_event` are fine). (This should also allow you to remove some noise from the `emit.rs` file, where you can inline a bunch of only-used-once functions.

Reply:
This has been addressed.